diff -up icewm-1.5.2/lib/keys.in.6~ icewm-1.5.2/lib/keys.in
--- icewm-1.5.2/lib/keys.in.6~	2019-03-13 20:14:32.718757710 +0100
+++ icewm-1.5.2/lib/keys.in	2019-03-13 20:16:09.556307666 +0100
@@ -9,9 +9,9 @@
 # You'll have to omit XK_ prefixs and to replace XF86XK_ prefixes by
 # XF86. Valid modifiers are Alt, Ctrl, Shift, Meta, Super and Hyper.
 #
-key "Alt+Ctrl+t"			xterm
+key "Alt+Ctrl+t"			xvt
 key "Alt+Ctrl+b"			xdg-open about:blank
-key "Alt+Ctrl+s"			xdg-open http://www.google.com
+key "Alt+Ctrl+s"			xdg-open http://yandex.com
 
 key "Super+KP_Subtract"			amixer sset Master 5%-
 key "Super+KP_Add"			amixer sset Master 5%+
@@ -29,8 +29,8 @@ key "XF86AudioLowerVolume"		amixer sset
 key "XF86AudioRaiseVolume"		amixer sset Master 5%+
 key "XF86AudioMute"			amixer sset Master toggle
 key "XF86HomePage"			xdg-open about:blank
-key "XF86Search"			xdg-open http://www.google.com
+key "XF86Search"			xdg-open http://yandex.com
 key "XF86Eject"				eject
-key "XF86Calculator"			/bin/sh -c "gnome-calculator || xcalc || ( type bc >/dev/null 2>&1 && xterm -e bc -l)"
+key "XF86Calculator"			/bin/sh -c "kcalc || gnome-calculator || xcalc || ( type bc >/dev/null 2>&1 && xvt -e bc -l)"
 
 switchkey "Super+p"			icewm-menu-xrandr
diff -up icewm-1.5.2/lib/toolbar.in.6~ icewm-1.5.2/lib/toolbar.in
--- icewm-1.5.2/lib/toolbar.in.6~	2019-02-04 21:05:00.000000000 +0100
+++ icewm-1.5.2/lib/toolbar.in	2019-03-13 20:14:32.718757710 +0100
@@ -4,6 +4,8 @@
 # since modifications to this file will be discarded when you
 # (re)install icewm.
 #
-prog xterm utilities-terminal xterm
-prog "Web browser" web-browser xdg-open about:blank
+prog Terminal terminals_section.png xvt
+prog Editor editors_section.png xeditor
+prog Browser web_browser_section.png www-browser
+prog MCC drakconf.png mcc
 
diff -up icewm-1.5.2/src/default.h.6~ icewm-1.5.2/src/default.h
--- icewm-1.5.2/src/default.h.6~	2019-03-13 20:14:32.718757710 +0100
+++ icewm-1.5.2/src/default.h	2019-03-13 20:18:12.210256612 +0100
@@ -171,7 +171,7 @@ XIV(int, netWorkAreaBehaviour,
 
 XSV(const char *, acpiIgnoreBatteries,          0)
 XSV(const char *, mailBoxPath,                  0)
-XSV(const char *, mailCommand,                  "xterm -name mutt -e mutt")
+XSV(const char *, mailCommand,                  "xvt -name mutt -e mutt")
 XSV(const char *, mailClassHint,                "mutt.XTerm")
 XSV(const char *, newMailCommand,               0)
 XSV(const char *, lockCommand,                  0)
@@ -179,7 +179,7 @@ XSV(const char *, clockCommand,
 XSV(const char *, clockClassHint,               "icewm.XClock")
 XSV(const char *, runDlgCommand,                0)
 XSV(const char *, openCommand,                  0)
-XSV(const char *, terminalCommand,              "xterm -hold")
+XSV(const char *, terminalCommand,              "xvt")
 XSV(const char *, logoutCommand,                0)
 XSV(const char *, logoutCancelCommand,          0)
 #if defined(__linux__)
@@ -196,7 +196,7 @@ XIV(int, taskBarCPUDelay,
 XIV(int, taskBarMEMDelay,                       500)
 XIV(int, taskBarNetSamples,                     20)
 XIV(int, taskBarNetDelay,                       500)
-XSV(const char *, cpuCommand,                   "xterm -name top -title Process\\ Status -e top")
+XSV(const char *, cpuCommand,                   "xvt -T Process\\ Status -e top")
 XSV(const char *, cpuClassHint,                 "top.XTerm")
 XIV(bool, cpuCombine,                           true)
 
diff -up icewm-1.5.2/src/yprefs.h.6~ icewm-1.5.2/src/yprefs.h
