diff -up icewm-1.5.2/src/yconfig.h.defaultfont icewm-1.5.2/src/yconfig.h
--- icewm-1.5.2/src/yconfig.h.defaultfont	2019-03-13 19:59:55.122094368 +0100
+++ icewm-1.5.2/src/yconfig.h	2019-03-13 20:09:49.045348840 +0100
@@ -30,23 +30,16 @@
 #ifndef __YCONFIG_H__
 #define __YCONFIG_H__
 
-#ifdef CONFIG_XFREETYPE
-#define FONT(pt) "-*-sans-medium-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define BOLDFONT(pt) "-*-sans-bold-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define TTFONT(pt) "-*-monospace-medium-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define BOLDTTFONT(pt) "-*-monospace-bold-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#else
 #ifdef FONTS_ADOBE
-#define FONT(pt) "-b&h-lucida-medium-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define BOLDFONT(pt) "-b&h-lucida-bold-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define TTFONT(pt) "-b&h-lucidatypewriter-medium-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define BOLDTTFONT(pt) "-b&h-lucidatypewriter-bold-r-*-*-*-" #pt "-*-*-*-*-*-*"
+#define FONT(pt) "-b&h-lucida-medium-r-*-*-*-" #pt "-*-*-*-*-*-*,-*--16-*"
+#define BOLDFONT(pt) "-b&h-lucida-bold-r-*-*-*-" #pt "-*-*-*-*-*-*,-*--16-*"
+#define TTFONT(pt) "-b&h-lucidatypewriter-medium-r-*-*-*-" #pt "-*-*-*-*-*-*,-*--16-*"
+#define BOLDTTFONT(pt) "-b&h-lucidatypewriter-bold-r-*-*-*-" #pt "-*-*-*-*-*-*,-*--16-*"
 #else
-#define FONT(pt) "-adobe-helvetica-medium-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define BOLDFONT(pt) "-adobe-helvetica-bold-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define TTFONT(pt) "-adobe-courier-medium-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#define BOLDTTFONT(pt) "-adobe-courier-bold-r-*-*-*-" #pt "-*-*-*-*-*-*"
-#endif
+#define FONT(pt) "-misc-fixed-medium-r-normal-*-*-" #pt "-*-*-*-*-16-*"
+#define BOLDFONT(pt) "-misc-fixed-bold-r-normal-*-*-" #pt "-*-*-*-*-16-*"
+#define TTFONT(pt) "-adobe-courier-medium-r-*-*-*-" #pt "-*-*-*-*-*-*,-*--16-*"
+#define BOLDTTFONT(pt) "-adobe-courier-bold-r-*-*-*-" #pt "-*-*-*-*-*-*,-*--16-*"
 #endif
 
 #define kfShift  1
