--- icewm-1.3-dev/src/ylocale.cc.pix	2006-08-07 19:19:43.000000000 +0200
+++ icewm-1.3-dev/src/ylocale.cc	2006-11-21 09:08:53.000000000 +0100
@@ -48,7 +48,7 @@
         fLocaleName = setlocale(LC_ALL, "C");
     }
 #warning "P1 should always use multibyte/fontset if I18N"
-    multiByte = (MB_CUR_MAX > 1);
+    multiByte = true;
 
     char const * codeset = NULL;
     int const codesetItems[] = { CONFIG_NL_CODESETS };
