--- icewm-1.3.0/src/wmapp.cc.pix	2007-08-10 22:40:17.000000000 +0200
+++ icewm-1.3.0/src/wmapp.cc	2007-08-10 23:40:24.000000000 +0200
@@ -1172,7 +1172,7 @@ YWMApp::YWMApp(int *argc, char ***argv, 
         break;
     default: /* click to focus */
         clickFocus = true;
-        focusOnAppRaise = false;
+        focusOnAppRaise = true;
         requestFocusOnAppRaise = true;
         raiseOnFocus = true;
         raiseOnClickClient = true;
