diff -up icewm-1.5.2/src/default.h.omv~ icewm-1.5.2/src/default.h
--- icewm-1.5.2/src/default.h.omv~	2019-03-13 20:14:08.876374319 +0100
+++ icewm-1.5.2/src/default.h	2019-03-13 20:14:15.454480172 +0100
@@ -5,7 +5,7 @@
 
 /************************************************************************************************************************************************************/
 XIV(bool, clickFocus,                           true)
-XIV(bool, focusOnAppRaise,                      false)
+XIV(bool, focusOnAppRaise,                      true)
 XIV(bool, requestFocusOnAppRaise,               true)
 XIV(bool, raiseOnFocus,                         true)
 XIV(bool, focusOnClickClient,                   true)
