diff -up icewm-1.5.2/src/yximage.cc.omv~ icewm-1.5.2/src/yximage.cc
--- icewm-1.5.2/src/yximage.cc.omv~	2019-03-13 21:47:13.821682522 +0100
+++ icewm-1.5.2/src/yximage.cc	2019-03-13 21:47:29.942921175 +0100
@@ -162,7 +162,8 @@ ref<YImage> YImage::load(upath filename)
 
 pstring YXImage::detectImageType(upath filename) {
      const int xpm = 9, png = 8, jpg = 4, len = max(xpm, png);
-     char buf[len+1] = {};
+     char buf[len+1];
+     memset(buf, 0, len+1);
      if (read_file(filename.string(), buf, sizeof buf) >= len) {
          if (0 == memcmp(buf, "/* XPM */", xpm)) {
              return ".xpm";
