---
 src/icewmbg.cc |    7 	6 +	1 -	0 !
 1 file changed, 6 insertions(+), 1 deletion(-)

Index: icewm-1.3.1/src/icewmbg.cc
===================================================================
--- icewm-1.3.1.orig/src/icewmbg.cc	2007-08-07 01:19:19.000000000 -0400
+++ icewm-1.3.1/src/icewmbg.cc	2009-08-17 05:16:54.000000000 -0400
@@ -277,7 +277,12 @@ void DesktopBackgroundManager::changeBac
             handleBackground = true;
         }
     } else if (DesktopBackgroundColor && DesktopBackgroundColor[0]) {
-        XSetWindowBackgroundPixmap(xapp->display(), desktop->handle(), 0);
+/* Removed by jmdault@mandrakesoft.com:
+ * This line made XFree crash! The program works nicely without it,
+ * but I haven't got a clue why... it's very weird!
+ * 
+						XSetWindowBackgroundPixmap(xapp->display(), desktop->handle(), 0);
+*/
         XSetWindowBackground(xapp->display(), desktop->handle(), bPixel);
         handleBackground = true;
     }
