--- icewm-1.3.0/src/yicon.cc.kdeicon	2007-06-20 13:14:39.000000000 +0200
+++ icewm-1.3.0/src/yicon.cc	2007-06-20 13:15:25.000000000 +0200
@@ -94,6 +94,18 @@ upath YIcon::findIcon(int size) {
         return fullpath;
 
 #if defined(CONFIG_IMLIB) || defined(CONFIG_GDK_PIXBUF_XLIB)
+    char *p;
+    char name[1024];
+    sprintf(icons_size, "%s", REDIR_ROOT(cs.c_str()));
+    p = strrchr(icons_size, '/');
+    strcpy(name, p ? p + 1 : icons_size);
+    /* support for KDE icon paths: /usr/share/icons/mini /usr/share/icons/large */
+    if (size == smallSize() || size == hugeSize()) {
+	sprintf(icons_size, "%s/%s", size == smallSize() ? "mini" : "large", name);
+	fullpath = findIcon(icons_size, size);
+	if (fullpath != null)
+	    return fullpath;
+    }
     sprintf(icons_size, "%s", REDIR_ROOT(cs.c_str()));
     fullpath = findIcon(icons_size, size);
     if (fullpath != null)
