--- src/base.h.gcc	2012-01-24 22:45:22.000000000 +0100
+++ src/base.h	2012-01-24 22:45:29.000000000 +0100
@@ -1,12 +1,6 @@
 #ifndef __BASE_H
 #define __BASE_H
 
-#if ( __GNUC__ == 3 && __GNUC_MINOR__ > 0 ) || __GNUC__ > 3
-#define deprecated __attribute__((deprecated))
-#else
-#define deprecated
-#endif
-
 /*** Atomar Data Types ********************************************************/
 
 #ifdef NEED_BOOL
--- src/yarray.h
+++ src/yarray.h
@@ -152,7 +152,7 @@ public:
     }
 
     virtual void remove(const typename YArray<DataType *>::SizeType index) {
-        if (index < YArray<DataType *>::getCount()) delete getItem(index);
+        if (index < YArray<DataType *>::getCount()) delete this->getItem(index);
         YArray<DataType *>::remove(index);
     }
     
