--- icewm-1.2.30/configure.in.old	2007-01-16 16:12:51.000000000 +0200
+++ icewm-1.2.30/configure.in	2007-01-16 16:13:31.000000000 +0200
@@ -820,15 +820,15 @@
     
       if test "${PKG_CONFIG}" != ""; then
 	GNOME_VER=2
-	GNOME2_CFLAGS=`pkg-config --cflags gnome-desktop-2.0`
-	GNOME2_LIBS=`pkg-config --libs gnome-desktop-2.0`
+	GNOME2_CFLAGS=`pkg-config --cflags gnome-desktop-2.0 libgnomeui-2.0`
+	GNOME2_LIBS=`pkg-config --libs gnome-desktop-2.0 libgnomeui-2.0`
 
 	AC_DEFINE(CONFIG_GNOME_MENUS, 1, [Define to make IceWM more GNOME-friendly])
 
 	APPLICATIONS="${APPLICATIONS} icewm-menu-gnome2"
 	GNOME2_PREFIX=`pkg-config --variable=prefix gnome-desktop-2.0`
 	GWMDIR="${GNOME2_PREFIX}/share/gnome/wm-properties/"
-	CONFIG_GNOME2_MENU_DIR="${GNOME2_PREFIX}/share/gnome/vfolders/"
+	CONFIG_GNOME2_MENU_DIR="${GNOME2_PREFIX}/share/desktop-directories/"
       fi
 
       if test "${GNOME2_CFLAGS}" = ""; then
