--- icewm-1.5.4/lib/keys.in.mgapref	2019-05-03 22:26:47.791676913 +0300
+++ icewm-1.5.4/lib/keys.in	2019-05-03 22:29:41.768184890 +0300
@@ -10,8 +10,8 @@
 # XF86. Valid modifiers are Alt, Ctrl, Shift, Meta, Super and Hyper.
 #
 key "Alt+Ctrl+t"			xterm
-key "Alt+Ctrl+b"			xdg-open about:blank
-key "Alt+Ctrl+s"			xdg-open https://www.google.com
+key "Alt+Ctrl+b"			www-browser about:blank
+key "Alt+Ctrl+s"			www-browser https://www.google.com
 
 key "Super+KP_Subtract"			amixer sset Master 5%-
 key "Super+KP_Add"			amixer sset Master 5%+
@@ -28,8 +28,8 @@
 key "XF86AudioLowerVolume"		amixer sset Master 5%-
 key "XF86AudioRaiseVolume"		amixer sset Master 5%+
 key "XF86AudioMute"			amixer sset Master toggle
-key "XF86HomePage"			xdg-open about:blank
-key "XF86Search"			xdg-open https://www.google.com
+key "XF86HomePage"			www-browser about:blank
+key "XF86Search"			www-browser https://www.google.com
 key "XF86Eject"				eject
 key "XF86Calculator"			/bin/sh -c "gnome-calculator || xcalc || ( type bc >/dev/null 2>&1 && xterm -e bc -l)"
 
--- icewm-1.5.4/lib/toolbar.in.mgapref	2019-04-23 23:12:00.000000000 +0300
+++ icewm-1.5.4/lib/toolbar.in	2019-05-03 22:26:47.791676913 +0300
@@ -4,6 +4,8 @@
 # since modifications to this file will be discarded when you
 # (re)install icewm.
 #
-prog xterm utilities-terminal xterm
-prog "Web browser" web-browser xdg-open about:blank
+prog Terminal terminals_section.png xvt
+prog Editor editors_section.png xeditor
+prog Browser web_browser_section.png www-browser
+prog MCC drakconf.png mcc
 
--- icewm-1.5.4/src/default.h.mgapref	2019-04-23 23:12:00.000000000 +0300
+++ icewm-1.5.4/src/default.h	2019-05-03 22:26:47.791676913 +0300
@@ -171,7 +171,7 @@
 
 XSV(const char *, acpiIgnoreBatteries,          0)
 XSV(const char *, mailBoxPath,                  0)
-XSV(const char *, mailCommand,                  "xterm -name mutt -e mutt")
+XSV(const char *, mailCommand,                  "xvt -T mutt -e mutt")
 XSV(const char *, mailClassHint,                "mutt.XTerm")
 XSV(const char *, newMailCommand,               0)
 XSV(const char *, lockCommand,                  0)
@@ -179,7 +179,7 @@
 XSV(const char *, clockClassHint,               "icewm.XClock")
 XSV(const char *, runDlgCommand,                0)
 XSV(const char *, openCommand,                  0)
-XSV(const char *, terminalCommand,              "xterm -hold")
+XSV(const char *, terminalCommand,              "xvt -hold")
 XSV(const char *, logoutCommand,                0)
 XSV(const char *, logoutCancelCommand,          0)
 #if defined(__linux__)
@@ -196,15 +196,15 @@
 XIV(int, taskBarMEMDelay,                       500)
 XIV(int, taskBarNetSamples,                     20)
 XIV(int, taskBarNetDelay,                       500)
-XSV(const char *, cpuCommand,                   "xterm -name top -title Process\\ Status -e top")
+XSV(const char *, cpuCommand,                   "xvt -T Process\\ Status -e top")
 XSV(const char *, cpuClassHint,                 "top.XTerm")
 XIV(bool, cpuCombine,                           true)
 
 #ifdef __linux__
-XSV(const char *, netCommand,                   "xterm -name 'ss' -title 'Socket Statistics' -hold -e sh -c 'which ss > /dev/null && watch -t ss -putswl || netstat -c'")
+XSV(const char *, netCommand,                   "xvt -T 'Socket Statistics' -hold -e sh -c 'which ss > /dev/null && watch -t ss -putswl || netstat -c'")
 XSV(const char *, netClassHint,                 "ss.XTerm")
 #else
-XSV(const char *, netCommand,                   "xterm -name netstat -title 'Network Status' -hold -e netstat -c")
+XSV(const char *, netCommand,                   "xvt -T 'Network Status' -hold -e netstat -c")
 XSV(const char *, netClassHint,                 "netstat.XTerm")
 #endif
 
