diff -urN icewm-1.2.14pre11/src/icewmbg_prefs.h icewm-1.2.14pre11-modified/src/icewmbg_prefs.h
--- icewm-1.2.14pre11/src/icewmbg_prefs.h	2004-03-16 21:19:23.000000000 +0100
+++ icewm-1.2.14pre11-modified/src/icewmbg_prefs.h	2004-03-24 12:17:02.226003100 +0100
@@ -3,12 +3,12 @@
 
 #include "yconfig.h"
 
-XSV(const char *, DesktopBackgroundColor, "rgb:00/20/40")
-XSV(const char *, DesktopBackgroundPixmap, 0)
+XSV(const char *, DesktopBackgroundColor, 0)
+XSV(const char *, DesktopBackgroundPixmap, "/usr/share/mga/backgrounds/default.jpg")
 XSV(const char *, DesktopTransparencyColor, 0)
 XSV(const char *, DesktopTransparencyPixmap, 0)
-XIV(bool, desktopBackgroundScaled, false)
-XIV(bool, centerBackground, false)
+XIV(bool, desktopBackgroundScaled, true)
+XIV(bool, centerBackground, true)
 XIV(bool, supportSemitransparency, true)
 
 void addBgImage(const char *name, const char *value, bool);
