diff -up ./src/wmmgr.cc.orig ./src/wmmgr.cc
--- ./src/wmmgr.cc.orig	2017-03-03 21:16:36.477147711 +0200
+++ ./src/wmmgr.cc	2017-03-03 21:17:12.327792713 +0200
@@ -1858,12 +1858,12 @@ void YWindowManager::restackWindows(YFra
 #ifndef LITE
     if (ctrlAltDelete && ctrlAltDelete->visible())
         count++;
+#endif
 
 #ifdef CONFIG_TASKBAR
     if (taskBar)
         count++;
 #endif
-#endif
 
     if (fLeftSwitch && fLeftSwitch->visible())
         count++;

