--- icewm-1.3-dev/src/yprefs.h.pix	2006-11-21 09:12:11.000000000 +0100
+++ icewm-1.3-dev/src/yprefs.h	2006-11-21 09:12:49.000000000 +0100
@@ -33,7 +33,7 @@
 XIV(bool, haveXft,                              true)
 #endif
 XSV(const char *, iconPath,                     "/usr/share/icons:/usr/share/icons/hicolor:/usr/share/icons/locolor:/usr/share/icons/crystalsvg:/usr/share/X11/icewm/icons")
-#define CONFIG_DEFAULT_THEME "icedesert/default.theme"
+#define CONFIG_DEFAULT_THEME "Galaxy/default.theme"
 XSV(const char *, themeName,                    CONFIG_DEFAULT_THEME)
 XSV(const char *, xineramaPrimaryScreenName,    0)
 
